### ISSUE TITLE
property to identify failures

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#1031]
I think we should reconsider whether there is a need to clearly identify publications that have failed to meet standards. We're relying on someone reading the summary, and a summary being present and clear on the matter.

Something like a "does not conform to" property that would establish what the publication was tested against could be useful if only for consistency and clarity of reporting...

**Comment by mattgarrish** [comment]
Looking at this again, it seems highly unlikely anyone would admit to this. Propose we close this one.
